Library/Formula/libiomp.rb

```diff
@@ -29,14 +21,13 @@ class Libiomp < Formula
   end
 
   def install
-    # fix makefile for yosemite:
-    inreplace "src/makefile.mk" do |s|
-      s.gsub! "`sw_vers -productVersion` > 10.6", "`sw_vers -productVersion` == '10.10' || `sw_vers -productVersion`> 10.6"
-    end
-
     intel_arch = MacOS.prefer_64_bit? ? "mac_32e" : "mac_32"
-
-    system "make", "compiler=clang"
+    args = std_cmake_args
+    args << "-DCMAKE_C_COMPILER=clang" << "-DCMAKE_CXX_COMPILER=clang++"
+    args << (MacOS.prefer_64_bit? ? "-Darch=32e" : "-Darch=32")
+    args << "-DCMAKE_BUILD_TYPE=Release"
```

Comment:
Ideally we'd like to avoid passing this as we already pass a `CMAKE_BUILD_TYPE`. Are every one of these CMake definitions above required for this to work?